### ISSUE TITLE
Request: Provide XLS BOM?

### ISSUE THREAD

**Issue by bepstein111** [zapta/legacy_stepper_motor_analyzer#2]
Hi there! This project looks great! My only request is to upload an XLS BOM so that people who don't want to source the individual surface mount components can just have the fab assemble that part.

Thanks for your hard work!

**Comment by zapta** [comment]
Will work on it. The BOM that Kicad generates is not that useful. I will create something manually. Not too many parts to list.

**Comment by zapta** [comment]
Added BOM.  https://github.com/zapta/simple_stepper_motor_analyzer/tree/master/kicad

**Comment by bepstein111** [comment]
Thanks!

**Comment by bepstein111** [comment]
Crap, I forgot, they also need a CPL (pick and place) file. If that's not easy for you to produce, a exported .zip from kicad (project, not gerber) will let me generate them in EasyEDA.

**Comment by zapta** [comment]
Ben, is this is what you are looking for?  https://support.jlcpcb.com/article/84-how-to-generate-the-bom-and-centroid-file-from-kicad

I can try going over the instructions and export.  BTW, I looked at SMD service at JLCPCB but am not sure they have the exact Alegro current sensors.

**Comment by bepstein111** [comment]
Yeah, that's it! Hmmm you checked on LCSC.com? That's unfortunate. I wonder if I'd have better luck through PCBway?

**Comment by coelholm** [comment]
@bepstein111 Did you have chance to generate that CPL file?

I found Alegro current sensor at [https://www.aliexpress.com/item/4000178092630.html](https://www.aliexpress.com/item/4000178092630.html)

**Comment by phil-barrett** [comment]
Be very careful of aliexpress vendors.  There is a reasonable chance you will get a fake chip. Personally, I would not trust them - way too many stories of remarked chips that aren't even close to what they claim to be.

**Comment by zapta** [comment]
Here is another one. This is witha  a different package that most makers
would consider easier to solder (will require a minor PCB change).

https://www.aliexpress.com/item/4000178147058.html

**Comment by zapta** [comment]
FYI, I just released an upgraded design which is based on the Raspberry Pi Pico. It also includes gerbers, bom and pos files for JLCPCB SMD service, for the parts they do carry.

https://github.com/zapta/simple_stepper_motor_analyzer
